tools/wake-format/emitter.cpp

```diff
@@ -1365,6 +1365,22 @@ wcl::doc Emitter::walk_binary(ctx_t ctx, CSTElement node) {
     MEMO_RET(*doc);
   }
 
+  // Lists with trailing unary commas should always exlode
+  // Ex:
+  //   a, b, c, => explode
+  //   a, b, c  => prefer flat when possible
+  if (parts.back().id() == CST_UNARY) {
```

Comment:
This code itself looks good to me. `back()` should always be valid because this is a binary op so you'll get at least 2, the case of no binary op and just a trailing comma will be un-exploded which is fine by me, all the checks look good to me.